core/nwb.file.yaml

```diff
@@ -258,10 +258,9 @@ groups:
         ADC boards, microscopes, etc.
       quantity: '?'
       groups:
-      - neurodata_type_def: Device
-        neurodata_type_inc: NWBContainer
-        doc: A data acquisition device, e.g. recording system.
-        quantity: '+'
+      - neurodata_type_inc: Device
+        doc: Data acquisition devices.
+        quantity: '*'
```

Comment:
Was there a reason why Device had quantity '+' (one or more)? I guess it makes sense that an NWB file should have some data and that data should have some device information attached to it. But then it means our barebones examples of making an NWBFile results in an invalid file (in practice, the validator says such a file is valid). 

I lean toward saying that an NWB file can be valid without any data and without any device.

Reply:
I agree

Reply:
I agree